prime-react/src/main/scala/lucuma/react/primereact/Toast.scala

```diff
@@ -33,7 +33,8 @@ case class ToastRef(
 case class Toast(
   position:   js.UndefOr[Toast.Position] = js.undefined, // default: TopRight
   modifiers:  Seq[TagMod] = Seq.empty,
-  baseZIndex: js.UndefOr[Int] = js.undefined
+  baseZIndex: js.UndefOr[Int] = js.undefined,
+  appendTo:   js.UndefOr[String] = js.undefined          // "self" or "body"
```

Comment:
Should we limit the values that can be passed here? The documentation says

> Valid values are any DOM Element and 'self'.
(https://primereact.org/toast/)

Does a string of "body" actually work? If so, neat, if not, since "self" is the default, we could maybe just allow DOM Elements?